bin/scancli

```diff
@@ -476,6 +476,18 @@ def displayhost_csv(fields, separator, nastr, dic, out=sys.stdout):
                         for line in utils.doc2csv(dic, fields, nastr=nastr)))
     out.write('\n')
 
+def _init_context(hostfilter, skip=None, limit=None):
```

Comment:
Since this is not a module, maybe it's not necessary to use the private function name convention here? But I'd like the name to be more explicit. Maybe `analyser_init_context`?